hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -1432,13 +1433,13 @@ protected final HoodieTable initTable(WriteOperationType operationType, Option<S
     }
 
     HoodieTable table;
-
-    this.txnManager.beginTransaction();
+    HoodieInstant ownerInstant = new HoodieInstant(true, CommitUtils.getCommitActionType(operationType, metaClient.getTableType()), instantTime.get());
```

Comment:
possible that instantTime is passed Option.empty() by the caller? For `initTable`, it's not likely but let's ensure that.